packages/openapi-ts/src/plugins/@tanstack/query-core/plugin.ts

```diff
@@ -1154,7 +1159,7 @@ export const handler: PluginHandler<
                       multiLine: true,
                       parameters: [
                         {
-                          name: 'options',
+                          name: 'localOptions',
```

Comment:
Could you rename this to `mutationOptions`?

```suggestion
                          name: 'mutationOptions',
```